src/nodes.coffee

```diff
@@ -3646,8 +3663,11 @@ exports.Parens = class Parens extends Base
     return @wrapInBraces fragments if @csxAttribute
     if bare then fragments else @wrapInParentheses fragments
 
-  ast: ->
-    @body.unwrap().ast()
+  astType: ->
```

Comment:
Again, completely disagree. It's so much more obvious what's going on (in one place) to just override `ast()` to forward to `@body.unwrap().ast()`

This is effectively the same thing that `Index` and `Access` are doing above, why didn't you insist on splitting up their `ast()` as well?

And I believe this would be incorrect as is, since you didn't forward `astLocationData()`, you'd be including the location data of the parens themselves in the generated AST, which is not what you want